extensions/wikia/AdEngine/js/lookup/rubiconFastlane.js

```diff
@@ -195,6 +195,7 @@ define('ext.wikia.adEngine.lookup.rubiconFastlane', [
 		rubicon.src = rubiconLibraryUrl;
 
 		node.parentNode.insertBefore(rubicon, node);
+		context = adContext.getContext();
```

Comment:
Hm, how bad will it be if we just leave `context,` instead of `context = adContext.getContext(),` in line #55 then? In other words: how about initiating context only in `call()`? :japanese_ogre: